polychrom/simulation.py

```diff
@@ -717,8 +762,8 @@ def do_block(
 
         if np.isnan(newcoords).any():
             raise IntegrationFailError("Coordinates are NANs")
-        if eK > self.eK_critical:
-            raise EKExceedsError("Ek={1} exceeds {0}".format(self.eK_critical, eK))
+        #if eK > self.eK_critical:
+        #    raise EKExceedsError("Ek={1} exceeds {0}".format(self.eK_critical, eK))
```

Comment:
please restore this feature (or provide justification why it should be removed)

Reply:
For Brownian Dynamics integrators, the notion of kinetic energy is somewhat meaningless since the particles do not have velocities, so this check is irrelevant. I added an if statement that only applies this check to non-BD integrators